src/Element/LNURLTip.tsx

```diff
@@ -42,10 +49,16 @@ export default function LNURLTip(props: LNURLTipProps) {
     const onClose = props.onClose || (() => { });
     const service = props.svc;
     const show = props.show || false;
-    const { note, author } = props
-    const amounts = [50, 100, 500, 1_000, 5_000, 10_000, 50_000];
+    const { note, author, target } = props
+    const amounts = [500, 1_000, 5_000, 10_000, 20_000];
+    const emojis: Record<number, string> = {
+      1_000: "👍",
+      5_000: "💜",
+      10_000: "😍",
+      20_000: "🤩"
```

Comment:
20k max? come on what about the 50k 100k?

Reply:
Yeah lemme add them,amounts is scrollable so no problem, gotta come up with emojis tho.

Reply:
🚀

Reply:
🍆 1M sats? 🤔

Reply:
Nah thats too far, maybe exploding head lol

Reply:
🤯